internal/storage/db/internal/db.go

```diff
@@ -383,25 +376,19 @@ func (s *dbStorage) Delete(ctx context.Context, ids ...namer.ModuleID) error {
 }
 
 func (s *dbStorage) ListPolicyIDs(ctx context.Context) ([]string, error) {
-	res, err := s.db.Select(goqu.C(PolicyTblKindCol), goqu.C(PolicyTblNameCol), goqu.C(PolicyTblVerCol)).
-		From(PolicyTbl).Executor().ScannerContext(ctx)
+	res, err := s.db.From(PolicyTbl).Select(goqu.L("CONCAT(LOWER(?),?,?,?,?)", goqu.C(PolicyTblKindCol), goqu.V("."), goqu.C(PolicyTblNameCol), goqu.V("."), goqu.C(PolicyTblVerCol))).Executor().ScannerContext(ctx)
 	if err != nil {
 		return nil, fmt.Errorf("could not execute %q query: %w", "ListPolicyIDs", err)
 	}
 	defer res.Close()
 
 	var policyKeys []string
 	for res.Next() {
-		var rec Policy
-		if err := res.ScanStruct(&rec); err != nil {
+		var rec string
+		if err = res.ScanVal(&rec); err != nil {
```

Comment:
I think you can use `ScanVals` to directly scan the whole resultset into the slice without iterating through the result set one by one.